api/app/models/resource.rb

```diff
@@ -247,6 +247,16 @@ def vimeo?
     external_video? && external_type == "vimeo"
   end
 
+  # @return [{ Symbol => Object }]
+  def packaging_metadata
+    metadata.with_indifferent_access
+      .merge(id: id)
+      .merge(slug: slug)
+      .merge(title: title_plaintext)
+      .merge(description: description_plaintext)
+      .merge(caption: caption_plaintext)
```

Comment:
I'm not gonna say we need to change this here, but in future, you can just do `.merge(id: id, slug: slug, title: title_plaintext, description: description_plaintext, caption: caption_plaintext)`